src/components/home/footer/style.tsx

```diff
@@ -11,25 +11,19 @@ const Footer = styled.div`
   background-color: #f8f9fb;
 `;
 
-const text = styled.p`
+export const text = styled.p`
```

Comment:
It's best practice to have component names to always start with a capital letter. This applies to all the component declarations below.

Reply:
Done  :white_check_mark: